Vulkan-Pipeline-Assistant/spirvresource.h

```diff
@@ -31,27 +31,242 @@ namespace vpa {
         }
     }
 
-    enum class SpirvResourceType {
-        INPUT_ATTRIBUTE, UNIFORM_BUFFER, STORAGE_BUFFER, SAMPLER_IMAGE, STORAGE_IMAGE, PUSH_CONSTANT
+    enum class SpvGroupname {
```

Comment:
Name should be in capitals to adhere with CamelCase.

Reply:
Fixed in latest